packages/apps/public/locales/fr/translation.json

```
@@ -600,26 +600,26 @@
+  "The account password as specified when creating the account. This is used to encrypt the backup file and subsequently decrypt it when restoring the account.": "Le mot de passe du compte tel que spécifié lors de la création du compte. Il est utilisé pour chiffrer le fichier de sauvegarde et ensuite le déchiffrer lors de la restauration du compte.",
   "The account that signed the input": "Le compte qui s ignoré l'entrée",
   "The account that will receive the funds": "Le compte qui recevra les fonds",
   "The account to which the proposed balance will be transferred if approved": "Le compte sur lequel le solde proposé sera transféré s'il est approuvé",
   "The account to which the tip will be transferred if approved": "Le compte sur lequel le pourboire sera transféré s'il est approuvé",
   "The account type (derivation) to use": "Le type de compte (dérivation) à utiliser",
-  "The account type that you wish to create. This is the top-level derivation.": "Le type de compte qui vous souhaitez créer. Il s'agit de la dérivation de niveau supérieur.",
+  "The account type that you wish to create. This is the top-level derivation.": "Le type de compte que vous souhaitez créer. Il s'agit de la dérivation de niveau supérieur.",
   "The account you want to claim to.": "Le compte sur lequel vous souhaitez percevoir les token.",
   "The account you want to register the preimage from": "Le compte à partir duquel vous souhaitez enregistrer la pré-image",
   "The account you want to register the proposal from": "Le compte à partir duquel vous souhaitez enregistrer la proposition",
-  "The account you will send funds from.": "Le compte que vous souhaitez envoyer des fonds depuis.",
+  "The account you will send funds from.": "Le compte depuis lequel vous souhaitez envoyer des fonds.",
   "The account's password specified at the creation of this account.": "Le mot de passe du compte spécifié lors de la création de ce compte.",
   "The actual JSONRPC module and function to make a call to.": "Le module JSONRPC réel et la fonction vers laquelle effectuer un appel.",
   "The actual account you wish to submit the bid with": "Le compte réel avec lequel vous souhaitez soumettre l'enchère",
   "The actual fully constructed signed output. This can be used for submission via other channels.": "La sortie signée réellement construite. Cela peut être utilisé pour la soumission via d'autres canaux.",
   "The actual proposal to make, based on the selected call and parameters thereof.": "La proposition réelle à faire, basée sur l'appel sélectionné et ses paramètres.",
   "The address for the deployed contract instance.": "L'adresse de l'instance de contrat déployée.",
-  "The address index (derivation on account) to use": "L'index d'adresse (dérivation sur compte) pour utiliser",
+  "The address index (derivation on account) to use": "L'indexe d'adresse (dérivation sur compte) à utiliser",
```

```suggestion
  "The address index (derivation on account) to use": "L'index d'adresse (dérivation sur compte) à utiliser",
```
It's like a database index https://fr.wikipedia.org/wiki/Index_(base_de_donn%C3%A9es)